packages/node_modules/@webex/widget-space/src/container.js

```diff
@@ -143,19 +144,55 @@ export class SpaceWidget extends Component {
       const callStartTime = (call?.activeParticipantsCount > 0) ? call?.startTime : null;
 
       let menuButton;
+      const {preferredWebexSite} = sparkInstance.meetings;
+      let isMeetButtonDisabled = false;
+      const meetings = sparkInstance.meetings.getAllMeetings();
+
+      if (Object.keys(meetings).length === 0) {
+        isMeetButtonDisabled = !preferredWebexSite;
+      }
+      else {
+        for (const key of Object.keys(meetings)) {
+          if (sparkInstance.meetings.getMeetingByType(CONVERSATION_URL, key)) {
+            isMeetButtonDisabled = false;
+            break;
+          }
+        }
+      }
+      let errorElementSpace = null;
+
+      if (isMeetButtonDisabled) {
+        const widgetError = {
+          temporary: false,
+          title: formatMessage(messages.noAbilityToStartMeeting)
+        };
+
+        errorElementSpace = (
+          <div className={classNames('webex-error-wrapper', styles.errorWrapper)}>
+            <ErrorDisplay
+              transparent={widgetError.temporary}
+              {...widgetError}
+            />
+          </div>
+        );
+      }
 
       if (activityTypes) {
         menuButton = [];
         activityTypes.forEach((at) => {
           if (at.name === 'meet') {
             menuButton.push((
-              <span className={classNames(`webex-tab-${at.name}`, styles.tabMeet)}>
+              <span
+                className={classNames(`webex-tab-${at.name}`, styles.tabMeet)}
+                title={isMeetButtonDisabled ? formatMessage(messages.noAbilityToStartMeeting) : undefined}
```

Comment:
How does title appear when button is enabled?

Reply:
when `isMeetButtonDisabled` is `true`, so when button is disabled we show the title, which is tooltip message.

Reply:
Is it possible that it would show `undefined` when `isMeetButtonDisabled` is `false`?

Reply:
No @rarajes2, if `isMeetButtonDisabled` is `false`, there won't be any `title` attr. 
<img width="572" alt="image" src="https://github.com/webex/react-widgets/assets/3918217/f8c448e2-07f5-4b68-bc6c-0fec91979f34">